ndcube/__init__.py

```diff
@@ -13,7 +13,7 @@
 
 __minimum_python_version__ = "3.6"
 
-__all__ = []
+__all__ = ['NDCube', 'NDCubeOrdered', 'NDCubeSequence']
```

Comment:
I suggest removing ```NDCubeOrdered``` as hasn't been given much attention and probably isn't recommended.